toc/rfc/rfc-draft-org-space-based-certificates-for-syslog-drains.md

```diff
@@ -0,0 +1,49 @@
+# Meta
+[meta]: #meta
+- Name: Support Org- and Space-Based Certificates for Loggregator Syslog TLS/mTLS Drains
+- Start Date: 2025-06-30
+- Author(s): ZPascal
+- Status: Draft
+- RFC Pull Request: [community#1229](https://github.com/cloudfoundry/community/pull/1229)
+
+
+## Summary
+
+This RFC proposes to introduce support for **organization- and space-scoped client certificates** for Cloud Foundry Loggregator syslog drains using mutual TLS (mTLS), covering both **HTTPS** and **syslog+TLS** protocols. By issuing certificates at the org or space level instead of per application, this initiative will simplify certificate lifecycle management, enable centralized rotation, and facilitate integration with central certificate authorities. The change targets the reduction of operational overhead and the enhancement of tenant-level security.
```

Comment:
Regarding "By issuing certificates at the org or space level instead of per application". Do you mean for the syslog drain use case? The https://docs.cloudfoundry.org/devguide/deploy-apps/instance-identity.html feature is used in different use cases.

Reply:
IMO, this is only for the certificates used for Syslog drains and has nothing to do with the instance identity certificates.